WaveAudioStream.cpp

```diff
@@ -132,66 +149,73 @@ WaveAudioStream::WaveAudioStream(File *file)
     riff_riff_t file_header;
     int length = file->read(&file_header, sizeof(file_header));
     if (length < 0) {
-        debug_printf("Error reading from file\n");
+        debug_printf("Error reading from file\r\n");
         return;
     }
     if (length != sizeof(file_header)) {
-        debug_printf("File not large enough\n");
+        debug_printf("File not large enough\r\n");
         return;
     }
     if (memcmp(&(file_header.identifier), "RIFF", 4) != 0) {
-        debug_printf("Incorrect header\n");
+        debug_printf("Incorrect header\r\n");
         return;
     }
     if (memcmp(&(file_header.format), "WAVE", 4) != 0) {
-        debug_printf("File is not a wave\n");
+        debug_printf("File is not a wave\r\n");
         return;
     }
-    debug_printf("RIFF data size = %lu\n",file_header.size);
+    debug_printf("RIFF data size = %lu\r\n",file_header.size);
 
 
     // Read chunks until format chunk is found
     riff_chunk_t current_chunk;
-    int result = read_chunk_info_skip(file, &current_chunk, "fmt ");
+    int result = read_chunk_info_skip(file, &current_chunk, "fmt ", 0, 0);
     if (result != 0) {
-        debug_printf("Problem reading \"fmt \" chunk\n");
+        debug_printf("Problem reading \"fmt \" chunk\r\n");
         return;
     }
+    
+    //Extensible has different size, so shouldn't return here
```

Comment:
It may be got to replace this check with a minimum size check. Maybe something like:
```c
if (current_chunk.size < 16) {
     debug_printf("\"fmt \" chunk is too small\r\n");
     return;
 }
```